operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet

```diff
@@ -12,15 +12,15 @@
           {
             alert: 'GitpodWorkspaceNodeHighNormalizedLoadAverage',
             labels: {
-              severity: 'warning',
+              severity: 'critical',
```

Comment:
@sagor999 can you change this to:

```
 labels: {
              severity: 'warning',
              team: 'workspace'
            },
```

That'll route here, https://gitpod.slack.com/archives/C03QZ1NH517, and give us a chance to measure if this is okay to promote to PagerDuty, etc.